src/highlight/highlight.controller.ts

```
@@ -0,0 +1,73 @@
+        return highlight;
+    }
+
+    @SkipThrottle()
+    @Get()
+    async findAll(
+        @Query('order') order?: Order,
+        @Query('limit') limit?: string,
+        @Query('lastEvaluatedKey') lastEvaluatedKey?: string // Note: Accepting as a string initially
+    ) {
+        const parsedLimit = limit ? parseInt(limit, 0) : undefined;
+        if (isNaN(parsedLimit ?? 0) && parsedLimit !== undefined) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)